src/modules/install.py

```diff
@@ -14,17 +15,13 @@
 # Load package information from versions.yaml
 def load_packages():
     """Load package configurations from versions.yaml file."""
-    yaml_path = os.path.join(
-        os.path.dirname(os.path.dirname(os.path.dirname(__file__))), "versions.yaml"
-    )
+    yaml_path = Path(__file__).resolve().parents[2] / "versions.yaml"
```

Comment:
It could be  a const like in the other one no ?

And let's not call it "yaml" again